root/static/scripts/relationship-editor/common/dialog.js

```diff
@@ -263,8 +263,14 @@ const RE = MB.relationshipEditor = MB.relationshipEditor || {};
                             if (gid === target.gid) {
                                 _.each(entity.displayableRelationships(vm)(), function (r) {
                                     switch (relationshipFilter) {
-                                      case 'same-entity-types': if (r.entityTypes !== relationship.entityTypes) { return; }; break;
-                                      case 'same-relationship-type': if (r.linkTypeID() !== relationship.linkTypeID()) { return; }; break;
+                                      case 'same-entity-types': if (r.entityTypes !== relationship.entityTypes) { 
+                                        return; 
+                                    }; 
+                                    break;
```

Comment:
I think these two lines (as well as perhaps some of the others below) should be indented slightly further so that it's clear that they are within the `switch` statement.

Reply:
I have added the indentation. Sorry about that.